src/main/java/Util/Parser.java

```diff
@@ -0,0 +1,10 @@
+package Util;
+
+import Tasks.Task;
+
+public class Parser {
```

Comment:
maybe can move the huge conditional that does the testing of input here

Reply:
So UI takes in parser as a variable and inputs queries to parser variable? All my show methods will be in Parser or UI then?